### ISSUE TITLE
"source" file

### ISSUE THREAD

**Issue by dbuezas** [Upcycle-Electronics/LGT8Fx-Datasheet#1]
Would be great if you added the source of the document you are creating. Maybe others can also help :)

**Comment by Upcycle-Electronics** [comment]
Yeah, I plan to add this with a proper repository. I've been trying to develop a working methodology. I also need to get a proper git toolchain working. I'm hoping to get the layered .xcf GIMP files I'm making compressed and uploaded too. If I can get those uploaded, and share the Tesseract-OCR and pdftk commands, hopefully others can make corrections as needed as I'm only using free software. I'm no expert, I just have lots of free time.
-Jake
